desktop/apps/article2/client/classic.js

```diff
@@ -1,7 +1,10 @@
 // Initializes all client-side Backbone views for "classic" layouts
 
-import $ from 'jquery'
+// import { $ } from 'jquery'
```

Comment:
If not using, can you remove? (Also, can just be `import $ from 'jquery'`

Reply:
Ooh whoops! I was testing something here -- will update